AUTHORS

```diff
@@ -1099,4 +1099,30 @@ Joannah Nanjekye <joannah.nanjekye@ibm.com>
 Ayush Kumar <ayushk7102@gmail.com>
 Kshitij <kshitijparwani.mat18@itbhu.ac.in>
 Daniel Hyams <dhyams@gmail.com>
+alijosephine <alijosephine@gmail.com>
 Matthias Köppe <mkoeppe@math.ucdavis.edu>
+mohajain <mohajain99@gmail.com>
+Anibal M. Medina-Mardones <ammedmar@gmail.com>
+Travis Ens <ens.travis@gmail.com>
+Evgenia Karunus <lakesare@gmail.com>
+RDxR10 <risirajdey@gmail.com>
+lastcodestanding <rohang71604@gmail.com>
+Andrey Lekar <andrey_lekar@adoriasoft.com>
+Abbas <42001049+iam-abbas@users.noreply.github.com>
```

Comment:
@iam-abbas do you want a different name and email address to be recorded here?

Reply:
Can we have email to iamabbas7@gmail.com

Reply:
Yes. Are you happy for your name to be recorded as just "Abbas" or would you prefer something else?

Reply:
Let's do `Abbas Mohammed`. Sorry for being in decisive ahaha. This is final.

Reply:
Thanks, I've updated that.